src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,69 @@
+@use "../utlis/variables";
+
+.stopwatch {
+  height: variables.$clock-size;
+  width: variables.$clock-size;
+  background-color: variables.$clock-face-color;
+  border-radius: variables.$clock-radius;
+  position: relative;
```

Comment:
```suggestion
.stopwatch {
  height: $clock-size;
  width: $clock-size;
  background-color: $clock-face-color;
  border-radius: $clock-radius;
  position: relative;
```

Reply:
Could you consider using the main file for imports? Imports should have the right order (variables and general page styling - first) It will allow you to use variables further in your files without additional `@use` and redundant code